### PR TITLE
DOCSP-21520: quick reference page

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-21520
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-21520-java-quick-reference/quick-reference/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/quick-reference.txt [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+Quick Reference
+===============
+
+.. default-domain:: mongodb
+
+This page shows the driver syntax for several MongoDB commands and links to
+their related reference and API documentation.
+
+TODO: this  assumes that you use the Filters package and convert output to
+JSON
```
Comment (terakilobyte) [inline source/quick-reference.txt:13]: issue: TODO?

File: source/quick-reference.txt [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+
+             coll.find(Filters.regex("title", "^Rocky")).skip(2);
+
+          .. output::
+             :language: json
+
+             [
+               { title: 'Rocky III', ... },
+               { title: 'Rocky IV', ... },
+               { title: 'Rocky V'}, ... }
```
Comment (terakilobyte) [inline source/quick-reference.txt:360]:
```suggestion
               { title: 'Rocky V', ... }
```

File: source/quick-reference.txt [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+                                    Filters.lt("year",  1970))));
+
+          .. output::
+             :language: json
+
+             AcknowledgedBulkWriteResult {
+               insertedCount: 1,
+               removedCount: ...,
+               ...
+             }
```
Comment (terakilobyte) [inline source/quick-reference.txt:224]: issue (non-blocking): Why include the output from this operation but not other operations like insert, update, and delete?

**Review by terakilobyte — APPROVED** [review]
lgtm with a few minor issues to fix

File: source/quick-reference.txt (hunk 1 above)
Comment (ccho-mongodb) [reply source/quick-reference.txt]: Whoops, didn't commit a file my last commit/push attempt

File: source/quick-reference.txt (hunk 3 above)
Comment (ccho-mongodb) [reply source/quick-reference.txt:221]: Seemed like something that some people ask about, seems reasonable to remove.

**Pull request merged**